test/server.ts

```diff
@@ -654,7 +654,7 @@ describe('server', function () {
             fastForward(10, 1000)
         })
 
-        it('should reply with a confirmable after an ack', function (done) {
+        it.skip('should reply with a confirmable after an ack', function (done) {
```

Comment:
Why do we skip this test?

Reply:
That was an unintended leftover, should now be removed :)